service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -118,6 +124,8 @@ public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
     // Store the run:
     UUID runId = UUID.randomUUID();
 
+    // is there a reason that this expression is here again? why not put it in a variable?
+    // e.g. String dataTableRowId = recordIds.get(0);
```

Comment:
Probably better as a tech talk or jira ticket rather than a comment in the code... I imagine finding this comment here in 10 years and wondering what to make of it.

(though FWIW... I believe Saloni's changes to iterate over the list rather than pick only the first will probably mean it's only like this temporarily)

Reply:
this was mostly a reminder to myself to raise it at tech talk -- which, actually, I failed to do

I'll remove it